GitUI/SplitterManager.cs

```diff
@@ -28,19 +28,19 @@ public void AddSplitter(SplitContainer splitter, string settingName, int? defaul
         public void RestoreSplitters()
         {
             // TODO: Disabled as splitters have become unstable. Refer to #8745 for more details.
```

Comment:
This comment should be deleted, too.

BTW: Never commit/merge code commented out using line comments (`////` in this case) again. Please use `#if false` ... `#endif` in order to get usable diffs.

Reply:
Aye, that's a good point to remember for all of us.
Thanks